exercise.wwwapi/Endpoints/LanguageEndpoints.cs

```diff
@@ -0,0 +1,54 @@
+﻿using exercise.wwwapi.Models;
```

Comment:
No endpoint for books.

Reply:
ok, will have a look at it

Reply:
I've included an interface, repository, and endpoint for books. However, I couldn't find any information about these in the requirements or readme file, so I initially wasn't aware they were necessary. Nevertheless, I've added them now, and it turned out to be a great practice.